Tests/SwiftDocCTests/Rendering/RESTSymbolsTests.swift

```diff
@@ -290,12 +290,6 @@ class RESTSymbolsTests: XCTestCase {
         } else {
             XCTFail("Unexpected attribute")
         }
-                
-        if case RenderAttribute.allowedValues(let values) = attributes[5] {
-            XCTAssertEqual(values, ["one", "two", "three"])
-        } else {
-            XCTFail("Unexpected attribute")
-        }
 
```

Comment:
question: Should this test verify that this information is displayed somewhere else on the page?